content/cloud/noops-sql-db.md

```diff
@@ -0,0 +1,120 @@
+title = "NoOps SQL Database"
```

Comment:
Can we refer to this feature as "NoOps SQLite Database"? It's important to note that it is a sqlite database.